Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN if [ "$http_proxy" != "" ]; \
     then \
       apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 \
```

Comment:
This would seem to be run only if an http_proxy arg is set to a nonempty string. How is the key added if there is no http_proxy set?

Reply:
That would be set via adding a build-arg argument to the docker build cli command invoked when building the image.

Reply:
I didn't change that logic, I just updated the key. If the problem was that this wasn't being executed at all, that would have been an issue before too.